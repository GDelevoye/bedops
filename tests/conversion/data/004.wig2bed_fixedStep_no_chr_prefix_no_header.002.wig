fixedStep chrom=3 start=400601 step=100 span=5
11
22
33
