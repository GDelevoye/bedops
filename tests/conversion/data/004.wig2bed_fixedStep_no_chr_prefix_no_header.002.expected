3	400600	400605	id-1	11.000000
3	400700	400705	id-2	22.000000
3	400800	400805	id-3	33.000000
